tests/general/InvalidLinkBearTest.py

```diff
@@ -74,72 +74,37 @@ def test_run(self):
         valid_file = """
         http://httpbin.org/status/200
         http://httpbin.org/status/201
-        https://en.wikipedia.org/wiki/200
-        """.splitlines()
-
-        self.assertResult(valid_file=valid_file)
-
-        # Link Redirect
-        invalid_file = """
-        http://httpbin.org/status/301
-        http://httpbin.org/status/302
-        """.splitlines()
-
-        self.assertResult(invalid_file=invalid_file)
-
-        # Invalid Link Not Found
-        valid_file = """
-        http://httpbin.org/status/401
-        """
-        invalid_file = """
-        http://httpbin.org/status/404
-        http://httpbin.org/status/410
-        """.splitlines()
-
-        self.assertResult(valid_file=valid_file, invalid_file=invalid_file)
-
-        # Invalid Link ServerError
-        invalid_file = """
-        http://httpbin.org/status/500
-        http://httpbin.org/status/503
-        """.splitlines()
-
-        self.assertResult(invalid_file=invalid_file)
+        http://httpbin.org/status/401  # Unauthorized
 
-        # Link Does Not Exist
-        invalid_file = """
-           http://coalaisthebest.com/
-        """.splitlines()
+        # Parentheses
+        https://en.wikipedia.org/wiki/Hello_(Adele_song)/200
 
-        self.assertResult(invalid_file=invalid_file)
+        # Quotes
+        "https://github.com/coala-analyzer/coala-bears/issues/200"
+        'http://httpbin.org/status/203'
 
-        # Test Regex
-        valid_file = """
-            https://en.wikipedia.org/wiki/Hello_(Adele_song)/200
-            "https://github.com/coala-analyzer/coala-bears/issues/200"
-            http://httpbin.org/status/200\n
-            http://httpbin.org/status/201
-            <http://httpbin.org/status/202>
-            'http://httpbin.org/status/203'
-            http://httpbin.org/status/204.....
-            http://not a link dot com
-        """.splitlines()
-
-        self.assertResult(valid_file=valid_file)
+        # Markup/down stuff
+        <http://httpbin.org/status/202>
+        http://httpbin.org/status/204.....
+        [httpbin](http://httpbin.org/status/200)
+        |http://httpbin.org/status/200|
 
-        # Markdown Links
-        valid_file = """
-            [httpbin](http://httpbin.org/status/200)
+        # Not a link
+        http://not a link dot com
```

Comment:
these are sometimes links. For instance, our university used to have this internal site like `http://something` without any GTLD or trailing slash or anything. When entered in the browser, they actually work because there is a internal DNS record that matches.